src/main/java/org/isf/patient/model/Patient.java

```diff
@@ -225,7 +226,7 @@ public Patient(Opd opd) {
 		
 		this.firstName = opd.getfirstName();
 		this.secondName = opd.getsecondName(); 
-		this.name = this.firstName + " " + this.secondName;
+		this.name = this.firstName + ' ' + this.secondName;
```

Comment:
1. Char is a single alphabet where as String is a sequence of characters. Char is primitive datatype where as String is a class.  Thus Strings get allocated and chars can share the same instance thus more "efficient".  I am of course talking about `' '` vs. `" "`.
2. Because that was the way the file was and I tried to not make major changes while increasing test coverage (sometimes I can't resist).   I plan on making another pass through the code looking for things like where `StringBuffer` should be used.

Reply:
OK, to a second pass IMHO.